packages/react-native/package.json

```diff
@@ -113,6 +113,7 @@
     "@jest/create-cache-key-function": "^29.6.3",
     "@react-native-community/cli": "14.0.0-alpha.2",
     "@react-native-community/cli-platform-android": "14.0.0-alpha.2",
+    "@react-native-community/cli-platform-apple": "14.0.0-alpha.2",
     "@react-native-community/cli-platform-ios": "14.0.0-alpha.2",
```

Comment:
FWIW I think I delete these entries / these entries are deleted in 0.76

Reply:
We're going to need `@react-native-community/cli-platform-apple` for `getDependencyConfig` and `getProjectConfig` though. This specific package will have to stay.

Reply:
Spoke offline. This stays.